doc/admin/troubleshooting.md

```diff
@@ -289,6 +289,40 @@ To remove the shared memory segment left behind by I/O Engine instance 1, issue:
 sudo ipcrm -M 0x10242049
 ```
 
+### Server Start Issues
+1. Read the log located in the `control_log_file`.
+1. Verify that the `daos_server` process is not currently running.
+1. Check the SCM device path in /dev.
+1. Verify the PCI addresses using `dmg storage scan`.
```

Comment:
1. prefix everywhere is intentional?

Reply:
Intentional, yes. Standard way of insuring correct sequence should an edit be made that would interrupt the numbering. This is standard for markdown but not required. It renders properly but I can change to sequential numbering if you prefer.